karate-core/src/main/java/com/intuit/karate/StepDefs.java

```diff
@@ -364,6 +371,18 @@ private void unprepare() {
         multiPart = null;
         request = null;
     }
+    private Object getRawResponse() {
```

Comment:
The return type from this method is ByteArrayInputStream and String. This is not consistent. We need to make it only one datatype.